index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./assets/style.css" type="text/css" rel="stylesheet">
```

Comment:
Do not use ./ to reference a file (styles or js), just use the folder name or file itself. (assets/style.css).
Change for all href's.